src/globals.c

```diff
@@ -584,10 +585,15 @@ static bool global_parse_opt(int opt, char *optarg)
 		set_content_url(optarg);
 		return true;
 	case 'P':
-		if (sscanf(optarg, "%ld", &update_server_port) != 1) {
+		if (sscanf(optarg, "%ld%c", &update_server_port, &tmp) != 1) {
```

Comment:
Because I'd like to check if there's something else trailing in the string. If I used '/0' the scanf would also return 1 if the user added a value like "123abc"

Reply:
but scanf isn't a good way on checking for overflows. I'll remove this call and use the strtoint_err funciton